api/account.py

```diff
@@ -1,128 +1,73 @@
 import sys
-from fastapi.exceptions import HTTPException
-from datetime import datetime
-from pydantic.main import BaseModel
-from starlette.responses import JSONResponse
-from api.user import get_user_by_token, get_user_exception
+
 sys.path.append("..")
 
+from pydantic.main import BaseModel
+from starlette.responses import JSONResponse
+from service.account import (
+    create_account_func,
+    delete_selected_account_func,
+    get_all_accounts_func,
+    get_removed_account_func,
+    restore_account_func,
+    update_account_func,
+)
+from service.auth import get_user_by_token
 from fastapi import APIRouter, Depends
 from pydantic.fields import Field
-from sqlalchemy.orm import Session
-from database import SessionLocal
-from models import models
-
-def get_db():
-    try:
-        db = SessionLocal()
-        yield db
-    finally:
-        db.close()
 
 router = APIRouter(
-    prefix="/accounts",
-    tags=["account"],
-    responses={401 : {"account" : "invalid"}}
+    prefix="/accounts", tags=["account"], responses={401: {"account": "invalid"}}
 )
 
+
 class Account(BaseModel):
-    price : int = Field(gt=0)
-    memo : str = Field(max_length=500)
+    price: int = Field(gt=0)
+    memo: str = Field(max_length=500)
 
 
 @router.post("")
-async def create_account(
-                        account: Account,
-                        user: dict = Depends(get_user_by_token),
-                        db: Session = Depends(get_db)):
-    if user is None:
-        raise get_user_exception()
-    
-    account_model = models.Account()
-    account_model.price = account.price
-    account_model.memo = account.memo
-    account_model.user_id = user.get("id")
+async def create_account(account: Account, user: dict = Depends(get_user_by_token)):
+    create_account_func(account.price, account.memo, user)
 
-    db.add(account_model)
-    db.commit()
-
-    return JSONResponse({"message" : "CREATE_SUCCESS"}, status_code=201)
+    return JSONResponse({"message": "CREATE_SUCCESS"}, status_code=201)
```

Comment:
```suggestion
    return JSONResponse({"message": "CREATE_SUCCESS"}, status_code=status.HTTP_201_CREATED)
```
p2: fastapi status 패키지 통해서 정의해 주세요. 그리고 실행 시 return 을 account 로 변경해 주세요.

Reply:
다른 부분의 status_code 도 모두 변경해 주세요.